src/lambda_codebase/moved_to_root.py

```diff
@@ -28,9 +29,10 @@ def worker_thread(sts, region, account_id, role, event):
         'remove_base')
 
     parameter_store = ParameterStore(region, role)
-    parameters = [param['Name'] for param in parameter_store.client.describe_parameters()['Parameters'] if 'Used by The AWS Deployment Framework' in param['Description']]
-    for parameter in parameters:
-        parameter_store.delete_parameter(parameter)
+    for page in paginator(parameter_store.client.describe_parameters):
```

Comment:
Could you please change this to use the Boto3 paginator instead?
See the docs in boto3 here: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html

Reply:
Absolutely, I'll update the PR next week!